### PR TITLE
TOC cleanup: API + Themes

### PR DESCRIPTION
**Pull request by bc-mkatz**
Ongoing tweaks to heading levels, to match TOC to legacy Dev Portal structure.

### CODE REVIEW

File: source/api/v2/index.html.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ includes:
   - api_objects_product_googleproductsearch
   - api_CRUD_products_googleproductsearch
   - api_objects_product_option
-  - api_CRUD_products_options
+  - api_CRUD_options.md
```
Comment (bc-AlyssNoland) [inline source/api/v2/index.html.md]: shouldn't include .md

File: source/includes/_api_CRUD_brands.md [hunk 2]
```diff
@@ -23,16 +23,20 @@ Filter parameters can be added to the URL query string to select specific brands
...
 
 | Parameter | Type | Example |
 | --- | --- | --- |
 | Page | int | /api/v2/brands?page={number} |
 | Limit | int | /api/v2/brands?limit={count} |
 
-```
+#### <span class="jumptarget"> Response </span>
+
+Example JSON returned in the response:
```
Comment (bc-AlyssNoland) [inline source/includes/_api_CRUD_brands.md]: I'm concerned that this won't carry over well in the current format when we want to go into three pane for API docs. You can see how TradeGecko handles this here: https://github.com/tradegecko/smaug/blob/master/source/index.html.md.erb#L68

File: source/api/v2/index.html.md (hunk 1 above)
Comment (bc-mkatz) [inline source/api/v2/index.html.md]: Good catches, and now fixed. But I can't commit the fixes without mixing in some unrelated Themes + CSS changes. Can you merge this now, and I'll commit + push the fixes on my next branch? BTW, these _do_ render with the trailing `.md` (my sloppy copy/paste error) – it's just superfluous.

File: source/includes/_api_CRUD_brands.md (hunk 2 above)
Comment (bc-mkatz) [inline source/includes/_api_CRUD_brands.md]: I think we've got a global issue here that we should discuss globally. Glad to hear that 3-pane is still planned as a future improvement. But we'll have to strip all these intro's back out when we do that. (Relatively straightforward search/replace in Sublime Text.) Whereas, the 2-column layout is incomprehensible without these cues.

**Pull request merged**